cmd/machine-config-daemon/start.go

```diff
@@ -41,6 +41,11 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 	// To help debugging, immediately log version
 	glog.Infof("Version: %+v", version.Version)
 
+	operatingSystem, err := daemon.GetRunningOS()
```

Comment:
Doesn't this need to happen after the `chroot` though? Otherwise, we'll be parsing the `/etc/os-release` of the container. Alternatively, might be beneficial to have a new e.g. `osrelease.NewAtRoot()` in general.

Reply:
You're 100% right. Updating.